sdk/include/opentelemetry/sdk/trace/samplers/README.md

```diff
@@ -0,0 +1,49 @@
+# Built-in Samplers
+The built-in samplers includes `AlwaysOn`, `AlwaysOff`, `Probability` and `ParentOrElse` samplers. This page explains relevant implementation details and how to use a sampler. For high-level design, please check out the [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#built-in-samplers).
+
+
+## TODOs
```

Comment:
I'd love if we could have issues instead of a `TODO` in a readme.

Reply:
Sounds good. I will move these to issues.